.github/PULL_REQUEST/new_config.md

```diff
@@ -0,0 +1,23 @@
+---
+name: New Config
+about: A new cluster config
+---
+
+Please follow these steps before submitting your PR:
+
+* [ ] If your PR is a work in progress, include `[WIP]` in its title
+* [ ] Its commits' summaries are reasonably descriptive
```

Comment:
I'm not sure we need this - we don't stipulate this anywhere else and I worry that it might scare newcomers (and this repo has quite a lot of git newcomers)

```suggestion
```